index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <div class="Container">
+     <div class="header">
+       <h1><a href="index.html">Welcome</a></h1>
+       <nav>
+       <img src="images/Enabled.png" alt="">
+      </nav>
+     </div>
+   
+
+   <div class="section">
```

Comment:
- [ ] The previous reviewer ask to use semantic tag here, you misunderstood the requirement. KIndly use like this:-
`<section class="section">`